### PR TITLE
Theme fix

### LINKED ISSUE #44: Align the theme
### Description

Align the theme of this website with the WasmEge website
For now changes will involve with Fonts and Color Sheme.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Aryanryn09**
## Fonts and Color Sheme updated

This commit updates the color scheme and font used throughout the application. These changes should improve the overall visual design and usability of the application, while also making it more consistent with modern design trends wrt the theme of this website with the WasmEge website

## Related issue

fixes #44 

Screenshot 2023-04-30 102543

### CODE REVIEW

**Comment by Bhardwaj-Himanshu** [comment]
Hi @Aryanryn09, 
Thank you so much for creating a pull request.
The changes are welcomed as well, but could you please also try attaching the screenshot of `npm run build` log with the pull request or `build the project locally` so that it does not throw a build error when merged as #46 
Thanks!

**Comment by Aryanryn09** [comment]
remote: Permission to WasmEdge/docs.git denied to Aryanryn09.
fatal: unable to access 'https://github.com/WasmEdge/docs.git/': The requested URL returned error: 403 i am getting this message  what can i do to get myself unblocked

**Comment by Bhardwaj-Himanshu** [comment]
Could you elaborate a bit more about the error message and why are you getting it?

Inshort what are you trying to do?

**Comment by Aryanryn09** [comment]
> Could you elaborate a bit more about the error message and why are you getting it?
> 
> Inshort what are you trying to do?

i think i've made and deleted multiple pr's at the same time

**Comment by Bhardwaj-Himanshu** [comment]
Are you trying to push to the original Wasmedge repository or your forked repo?

You won't be having the permissions to push to the Wasmedge original docs repo.

**Comment by alabulei1** [comment]
@Aryanryn09  LGTM! Could you fix the conflicts first? Sorry for my late reply. Because I was busy filling in more content to this doc.

**Comment by Aryanryn09** [comment]
> @Aryanryn09 LGTM! Could you fix the conflicts first? Sorry for my late reply. Because I was busy filling in more content to this doc.

Sure

File: docs/develop/wasmedge/features.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ Through the [WasmEdge-Quickjs](https://github.com/second-state/wasmedge-quickjs)
...
 
-Option #1 is to [use an OCI runtime crun](../deploy/oci-runtime/crun) (the C version of runc — mainly supported by Red Hat). crun decides whether an OCI image is Wasm or Linux based on image annotations. If the image is annotated as wasm32, crun will bypass the Linux container setup and just use WasmEdge to run it. If the image is annotated as wasm32, crun will bypass the Linux container setup and use WasmEdge to run the image. By using crun, you can get the entire Kubernetes stack - including CRI-O, containerd, Podman, kind, micro k8s, and k8s - to work with Wasm images.
 
-Option #2 is to [use a containerd-shim to start Wasm "containers" via runwasi](../deploy/cri-runtime/containerd.md). Basically, containerd could look at the image's target platform. It uses runwasi if the image is wasm32 and runc if it is x86 / arm. This is the approach used by Docker + Wasm.
```
Comment (alabulei1) [inline docs/develop/wasmedge/features.md:48]: I think we should keep Option #2.

**Review by alabulei1 — APPROVED** [review]

File: docs/develop/wasmedge/features.md [hunk 2]
```diff
@@ -41,11 +41,11 @@ Through the [WasmEdge-Quickjs](https://github.com/second-state/wasmedge-quickjs)
 
 WasmEdge could be seamlessly integrated with the existing cloud-native infra.
 
-To integrate WasmEdge with your existing cloud-native infrastructure, there are several options available for managing Wasm applications as "containers" under Kubernetes. These options enables you to run Linux containers and Wasm containers side by side within a Kubernetes cluster.
+There are several options to manage Wasm apps as "containers" under Kubernetes. All options will give you a Kubernetes cluster that runs Linux containers and WasmWasm containers side by side.
```
Comment (adithyaakrishna) [inline docs/develop/wasmedge/features.md:44]:
```suggestion
There are several options to manage Wasm apps as "containers" under Kubernetes. All options will give you a Kubernetes cluster that runs Linux containers and Wasm containers side by side. 
```

File: src/css/custom.css [hunk 3]
```diff
@@ -6,25 +6,31 @@
...
-  --ifm-color-primary-light: #29d5b0;
-  --ifm-color-primary-lighter: #32d8b4;
-  --ifm-color-primary-lightest: #4fddbf;
+  --ifm-color-primary: #d693e1;
+  --ifm-color-primary-dark: #c97fd6;
+  --ifm-color-primary-darker: #b56bca;
+  --ifm-color-primary-darkest: #a357be;
+  --ifm-color-primary-light: #e1b6ec;
+  --ifm-color-primary-lighter: #e8c3f0;
+  --ifm-color-primary-lightest: #f2dfff;
```
Comment (adithyaakrishna) [inline src/css/custom.css:34]: These changes are also in another PR, #51 Can we remove those changes from this PR and focus on only one change i.e. to the `features.md` file?

**Comment by adithyaakrishna** [comment]
@Aryanryn09 Could you please let us know if you have any updates on this PR?

**Comment by alabulei1** [comment]
Since @Aryanryn09 didn't update this PR, I will close this PR first.

If you're ready to update the PR, please reopen it. Thanks.

**Pull request closed without merging**